src/prefect/settings.py

```diff
@@ -583,6 +584,20 @@ def default_cloud_ui_url(settings, value):
 )
 """The maximum size in bytes for a single log."""
 
+PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW = Setting(
+    Literal["warn", "error", "ignore"],
+    default="warn",
+)
+"""
+Controls the behavior when logs are attempted to be sent to Orion without a flow run id.
```

Comment:
```suggestion
Controls the behavior when logs are sent to Orion without a flow run id.
```

Reply:
I also want better phrasing than "attempted to be sent," but I don't believe just "sent" would be accurate... I'll change it to "when loggers attempt to send logs to". Thanks!

Reply:
I'm okay with that. Thanks